packages/ndla-ui/src/ProgrammeV2/ProgrammeV2.stories.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import ProgrammeV2 from './ProgrammeV2';
+import { programmeV2 } from '../../../../dummydata/mockPrograms';
+import { defaultParameters } from '../../../../stories/defaults';
+
+export default {
+  title: 'PATTERNS/ProgrammeV2',
```

Comment:
Det vises med store bokstaver, men det er css.
```suggestion
  title: 'Patterns/ProgrammeV2',
```